### PR TITLE
Added login and registration page using webjar.

### PR DESCRIPTION
**Pull request by LauraMachado21**
## Issue

#33 

## Description of changes
Added the following pages using webjars sb-admin-2
- Login
- Registration
- Admin registration
- Forgot-password
- 404 page

## Screenshots (If Applicable)

Screen Shot 2019-03-18 at 9 26 32 PM
Screen Shot 2019-03-18 at 9 26 37 PM
Screen Shot 2019-03-18 at 9 26 40 PM
Screen Shot 2019-03-18 at 9 26 54 PM

## Checklist
- [x] I have unit tested my code.
- [ ] I have integration tested my code.
- [x] All new and existing tests passed.
- [ ] I have [updated the diagrams](https://github.com/amazin-team/Amazin-online-bookstore/wiki/Updating-the-diagrams) ModelsUMLClassDiagram and EntityRelationshipDiagram (if necessary).

### CODE REVIEW

**Review by zaidoon1 — APPROVED** [review]
This looks really good, nice work!

File: src/main/resources/public/error/404.html [hunk 1]
```diff
@@ -1,21 +1,87 @@
...
-        <nav>
-            <a href="/">Go to homepage</a><br>
-        </nav>
+        <!-- End of Main Content -->
+
+        <!-- Footer -->
+        <footer class="sticky-footer bg-white">
+            <div class="container my-auto">
+                <div class="copyright text-center my-auto">
+                    <span>Copyright &copy; Your Website 2019</span>
```
Comment (abungay) [inline src/main/resources/public/error/404.html]: change 'Your Website' to 'Amazin'

**Comment by abungay** [comment]
Looks really nice. Love how with webjars we don't have to add many extra files to our repo and I'm a big fan of this SB-admin-2 template and the pugs too! 😃

**Comment by abungay** [comment]
Admin registration can be found at /admin/register and contains 1 extra field in the form for registration token.

**Review by abungay — APPROVED** [review]

**Pull request merged**